src/SFA.DAS.EmployerAccounts.Events/Messages/Message.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace SFA.DAS.EmployerAccounts.Events.Messages
+{
+    public abstract class Message
```

Comment:
I think the message base class should be place in the DAS shared project message library as its something that could be commonly used by all. Only the created date field would be in it however and it should be renamed to MessageBase as its a base class.

Reply:
There is already a message base class in shared messages. The current EAS messages do not use that, instead not have a common class. I've removed signed by name because it might not be used in all future files....having said that it's not being agile to look into the future...I should remove SignedByName when it becomes necessary to do so.

Reply:
The reason I mentioned about removing it was if it was moved to the shared project message project. You don't want to leak domain references in shared projects if you can help it. Same with Account ID. If you keep it in the MA domain these can remain but it might be useful to place the more generic stuff in the shared project message project as others might find it useful. We have done this with the message handling code so it's use it linked to us doing that with regards to the agile side.